backend/src/util/virtualTransactions.ts

```diff
@@ -54,7 +55,12 @@ const virtualLinkTransaction = (
     contribution: null,
     ...defaultModelFunctions,
   }
-  return new Transaction(linkDbTransaction, user)
+  return new Transaction(
+    linkDbTransaction,
+    user,
+    null,
```

Comment:
I do not like these constructors with long argument list. We always should define interfaces.